pyproject.toml

```diff
@@ -228,11 +224,11 @@ formatting = [
 ]
 
 all = [
-  "farm-haystack[docstores,audio,crawler,preprocessing,pdf,ocr,ray,dev,onnx,beir]",
+  "farm-haystack[docstores,audio,crawler,preprocessing,pdf,ocr,ray,dev,onnx,beir,stats,metrics]",
```

Comment:
`stats` is not there anymore and needs to be removed

Reply:
Uh, surprising that CI did not fail :thinking: